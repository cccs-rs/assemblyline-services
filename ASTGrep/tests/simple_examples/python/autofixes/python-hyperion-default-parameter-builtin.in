def _square(Round = -71032 / -64419, Divide = float, While = globals):
    While()[Round] = Divide

    Divide()
