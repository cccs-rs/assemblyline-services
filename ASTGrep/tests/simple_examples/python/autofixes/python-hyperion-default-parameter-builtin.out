def _square(Round = 1.10266, Divide = float, While = globals):
    globals()[Round] = Divide

    Divide()
